### ISSUE TITLE
Logging installation fails on containerized installation

### ISSUE THREAD

**Issue by juanwolf** [openshift/openshift-ansible#7151]
#### Description

I tried to install the logging platform into openshift using the `playbooks/byo/openshift-cluster/openshift-logging.yml` playbook, but it failed with this error: 
```
 FAILED! => {"changed": false, "cmd": "oc get pod -l component=es,provider=openshift -n logging -o ''", "msg": "[Errno 2] No such file or directory", "rc": 2}
``` 

##### Version

```
ansible 2.4.3.0
  config file = /home/ubuntu/ansible/openshift-ansible/ansible.cfg
  configured module search path = [u'/home/ubuntu/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]

openshift-ansible-3.7.29-1
```

##### Steps To Reproduce
1. Setup a master "containerized"
2. Run the `playbooks/byo/openshift-cluster/openshift-logging.yml` playbook

##### Expected Results
Describe what you expected to happen.

```
The playbook ends successfully
```

##### Observed Results

```
TASK [openshift_logging_elasticsearch : command] ******************************************
fatal: [xxxx.xxxx.xxxx.xxxx]: FAILED! => {"changed": false, "cmd": "oc get pod -l component=es,provider=openshift -n logging -o ''", "msg": "[Errno 2] No such file or directory", "rc": 2}

```

To be a bit more detailed the actual command failing is in: `roles/openshift_logging_elasticsearch/tasks/get_es_version.yml:2`

##### Additional Information

```
OS: CentOS atomic. Version: 7.1712.1 (2018-01-12 10:25:04)
```

**Comment by vrutkovs** [comment]
Seems it would be fixed by #7081

**Comment by juanwolf** [comment]
I
